drivers/timer/sam0_rtc_timer.c

```diff
@@ -29,8 +29,13 @@
 #endif
 
 /* Number of sys timer cycles per on tick. */
+#ifdef SOC_ATMEL_SAM0_RTC_FREQ_HZ
+#define CYCLES_PER_TICK (SOC_ATMEL_SAM0_RTC_FREQ_HZ \
+			 / CONFIG_SYS_CLOCK_TICKS_PER_SEC)
+#else
```

Comment:
Is this related to something on line 28?

Reply:
The HW cycles and RTC cycles are derived from different clock generators.